tools/main.cpp

```diff
@@ -224,6 +224,8 @@ int main(int argc, char **argv)
         // deprecated - some use undocumented short options to catch them as such
         { "border", required_argument, 0, 'b' },
         { "ignore-layout", no_argument, 0, 'i' },
+        { "no-footer", no_argument, 0, 'y' },
```

Comment:
We can group them under a 'd' shortcut, which we will keep for deprecating other options in the future. (You should get the long option name with `long_options[option_index].name`). With this we avoid to arbitrary shortcuts that did not exist before for deprecating options